app/Browsershot/BrowsershotFactory.php

```diff
@@ -41,7 +42,7 @@ public function setSize(int $width, int $height): self
     public function setFileExtension(string $extension = 'png'): self
```

Comment:
You could use the constant here as well:
```suggestion
    public function setFileExtension(string $extension = FileInterface::IMAGE_PNG): self
```